bin/xml/impl/glassfish/jersey.xml

```diff
@@ -1,6 +1,6 @@
 <!--
 
-    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2018, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is the copyright update policy to update even the untouched files with 2020 ?
In my understanding only the files which were modified recently need to have these updates.